services/galley/test/integration/what-to-test

```diff
@@ -0,0 +1,7 @@
+For federation, what do we want to test?
+
+Lets start with conversations:
+
+1. conversation list might have remote conversations
```

Comment:
Is the plan to create these conversation in the DB directly or do we want to go through `joinConversationById`?